src/components/checkbox/checkbox.scss

```diff
@@ -250,6 +250,10 @@ $_md-checkbox-indeterminate-checked-easing-function: cubic-bezier(0.14, 0, 0, 1)
   }
 }
 
+.md-checkbox-layout label {
+  line-height: 24px;
```

Comment:
The label looks a bit high without it.  I noticed radio buttons were using line-height, so tried to make it consistent.